hamocc/mo_bgcmean.F90

```diff
@@ -2650,7 +2651,7 @@ subroutine msksrf(pos,idepth)
     !
     ! Local variables
     integer :: i,j,l
-    real, parameter :: mskval=nf90_fill_double
+    real(rp), parameter :: mskval=nf90_fill_double
```

Comment:
Here is a bit the question how to handle this. 

If `rp==4` one would need to select also ` nf90_fill_float` 

Maybe issue and error if rp is neither 4 nor 8?